### DISCUSSION TITLE
Multiple Source Folders for single DB page

### DISCUSSION THREAD

**Discussion by Syrkres** [RafaelGB/obsidian-db-folder#837] [Ideas]
As I mentioned in my previous idea is that I have over 5000 notes in a single folder. This is a collection of NPCs(characters) which I use for my world. Obviously 5000 is a lot for a single folder and it would be nice to be able to break these down into multiple folders.

For example I could have a folder of Dwarves, one for Elves, one for Humans, then have a single NPC DB page which pulls all the notes from the different folders into a single page.

**Comment by hollowhemlock** [comment]
Have you tried the following folder structure:

- characters (this is your db folder)
    - Dwarves
    - Elves
    - Humans
